### ISSUE TITLE
[Accessibility][Product Carousel] Decorative carousel product images are not marked up as such

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1766]
**Steps to Reproduce**
1. With NVDA operating, navigate to items in carousel using the TAB key.

**Current Behavior**
Carousel product images lack an alt attribute, so users of assistive technology are aware of the presence of the images but not the content. For example, for the "Ingrid Running Jacket" item in the carousel, NVDA users hear "wj04-orange_main_2 Unlabeled graphic Ingrid Running Jacket-XS-Orange$84.00 link".

**Expected behavior/code**
If no descriptive textual alternative is available for the image that would not duplicate the text that follows it, the image should be marked as decorative so assistive technology can ignore it.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users

**WCAG SC**
1.1.1 Non-text Content

**Code snippet**
```
<img class="product__image" src="/content/dam/core-components-examples/library/cif-sample-assets/catalog/product/ingrid-running-jacket/wj04-orange_main_2.jpg" data-gallery-role="currentimage">
```

**Possible Solution**
To identify an image as decorative, provide a null alt attribute (alt="").

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by mhaack** [comment]
Already fixed by https://github.com/adobe/aem-core-cif-components/pull/663
